### PR TITLE
TRUNK-5134: Replace path constructions using separator char

### PR DESCRIPTION
**Pull request by jessieanker**
## Description of what I changed
 liquibase package changes for replacing path constructions using separator char

## Issue I worked on 
see https://issues.openmrs.org/browse/TRUNK-5134

## Checklist: I completed these to help reviewers :)
 

- [x ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: liquibase/src/main/java/org/openmrs/liquibase/Main.java [hunk 1]
```diff
@@ -9,23 +9,24 @@
...
+import java.nio.file.Paths;
+
 import org.dom4j.DocumentException;
 
 public class Main {
 	
-	public static final String LIQUIBASE_CORE_DATA_SOURCE_PATH = "." + File.separator + "snapshots" + File.separator
-	        + "liquibase-core-data-SNAPSHOT.xml";
+	public static final String LIQUIBASE_CORE_DATA_SOURCE_PATH = Paths
+	        .get(".", "snapshots", "liquibase-core-data-SNAPSHOT.xml").toString();
```
Comment (HerbertYiga) [inline liquibase/src/main/java/org/openmrs/liquibase/Main.java:20]: thanks @jessieanker   looking good

File: liquibase/src/main/java/org/openmrs/liquibase/Main.java [hunk 2]
```diff
@@ -9,23 +9,24 @@
  */
 package org.openmrs.liquibase;
 
-import java.io.File;
 import java.io.IOException;
+import java.nio.file.Paths;
+
```
Comment (HerbertYiga) [inline liquibase/src/main/java/org/openmrs/liquibase/Main.java:14]: just some little unwanted spaces

File: liquibase/src/test/java/org/openmrs/liquibase/AbstractSnapshotTunerTest.java [hunk 3]
```diff
@@ -9,22 +9,22 @@
...
+import static org.hamcrest.Matchers.equalToCompressingWhiteSpace;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+
```
Comment (HerbertYiga) [inline liquibase/src/test/java/org/openmrs/liquibase/AbstractSnapshotTunerTest.java:23]: and here

File: liquibase/src/test/java/org/openmrs/liquibase/AbstractSnapshotTunerTest.java [hunk 4]
```diff
@@ -9,22 +9,22 @@
  */
 package org.openmrs.liquibase;
 
-import java.io.File;
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalToCompressingWhiteSpace;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
```
Comment (HerbertYiga) [inline liquibase/src/test/java/org/openmrs/liquibase/AbstractSnapshotTunerTest.java:17]: unwanted space

File: liquibase/src/test/java/org/openmrs/liquibase/CoreDataTunerTest.java [hunk 5]
```diff
@@ -9,13 +9,20 @@
...
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 import java.io.IOException;
 import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
+
```
Comment (HerbertYiga) [inline liquibase/src/test/java/org/openmrs/liquibase/CoreDataTunerTest.java:25]: no need of this

File: liquibase/src/test/java/org/openmrs/liquibase/SchemaOnlyTunerTest.java [hunk 6]
```diff
@@ -9,12 +9,20 @@
...
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+
 import java.io.IOException;
 import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
+
```
Comment (HerbertYiga) [inline liquibase/src/test/java/org/openmrs/liquibase/SchemaOnlyTunerTest.java:25]: no need of this

File: liquibase/src/main/java/org/openmrs/liquibase/Main.java (hunk 2 above)
Comment (teleivo) [reply liquibase/src/main/java/org/openmrs/liquibase/Main.java:14]: @HerbertYiga that's alright. java imports are grouped at the top, then javax, then org, then com. see https://wiki.openmrs.org/display/docs/Install+and+Configure+Eclipse+For+OpenMRS+Development#InstallandConfigureEclipseForOpenMRSDevelopment-ConfigureJavaImportOrder

File: liquibase/src/test/java/org/openmrs/liquibase/AbstractSnapshotTunerTest.java (hunk 4 above)
Comment (teleivo) [reply liquibase/src/test/java/org/openmrs/liquibase/AbstractSnapshotTunerTest.java:17]: That's ok. See my first comment

File: liquibase/src/test/java/org/openmrs/liquibase/AbstractSnapshotTunerTest.java (hunk 3 above)
Comment (teleivo) [reply liquibase/src/test/java/org/openmrs/liquibase/AbstractSnapshotTunerTest.java:23]: Also ok

File: liquibase/src/test/java/org/openmrs/liquibase/CoreDataTunerTest.java (hunk 5 above)
Comment (teleivo) [reply liquibase/src/test/java/org/openmrs/liquibase/CoreDataTunerTest.java:25]: Also ok. See first comment

File: liquibase/src/test/java/org/openmrs/liquibase/SchemaOnlyTunerTest.java (hunk 6 above)
Comment (teleivo) [reply liquibase/src/test/java/org/openmrs/liquibase/SchemaOnlyTunerTest.java:25]: Again, this is fine. See first comment

**Review by teleivo — APPROVED** [review]
Thank you @jessieanker ! Looks good!

**Pull request merged**